README.rst

```diff
@@ -73,6 +73,22 @@ Some Python HTTP clients that you can use to retrieve HTML include `requests`_,
 .. _urllib.request module: https://docs.python.org/3/library/urllib.request.html
 
 
+Content removal
```

Comment:
I've adjusted this title from `Test data` to `Content removal` because that seems to be what I've ended up focusing on -- and if so, we should be direct and get to the point, because for people concerned about their information being included here, it's probably most important for them to get to describe what we can do about it.